src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -244,17 +245,47 @@ void Profile::LayerJson(const Json::Value& json)
 }
 
 // Method Description:
+// - Returns this profile's icon path, if one is set. Otherwise returns the
+//   empty string. This method will expand any environment variables in the
+//   path, if there are any.
+// Return Value:
+// - this profile's icon path, if one is set. Otherwise returns the empty string.
+winrt::hstring Profile::ExpandedIconPath() const
+{
+    if (_IconPath.empty())
+    {
+        return _IconPath;
+    }
+    winrt::hstring envExpandedPath{ wil::ExpandEnvironmentStringsW<std::wstring>(_IconPath.c_str()) };
+    return envExpandedPath;
+}
+
+// Method Description:

```

Comment:
This looks like its from a bad merge?

Reply:
Yea, I figured it was from a different feature, and left it. Removed and still works. Thanks for all the help. Any suggestion on a new issue?